### PR TITLE
doc ICD issue for autoscaling

### PR DESCRIPTION
**Pull request by SirSpidey**
### Description

Addresses https://github.com/terraform-ibm-modules/terraform-ibm-icd-postgresql/issues/52

#### No release required

- [ ] Examples or tests (addition or updates of examples or tests)
- [x] Documentation update
- [ ] CI-related update (pipeline, etc.)
- [ ] Other changes that don't affect Terraform code

#### Release required

- [ ] Bug fix (patch release (`x.x.X`): Change that fixes an issue and is compatible with earlier versions)
- [ ] New feature (minor release (`x.X.x`): Change that adds functionality and is compatible with earlier versions)
- [ ] Breaking change (major release (`X.x.x`): Change that is likely incompatible with previous versions)

##### Release notes content

Replace this text with information that users need to know about the bug fixes, features, and breaking changes. This information helps the merger write the commit message that is published in the release notes for the module.

---

### Checklist for reviewers

- [ ] If relevant, a test for the change is included or updated with this PR.
- [ ] If relevant, documentation for the change is included or updated with this PR.

### Merge actions for mergers

- Merge by using "Squash and merge".
- Use a relevant [conventional commit](https://www.conventionalcommits.org/) message that is based on the PR contents and any release notes provided by the PR author.

    The commit message determines whether a new version of the module is needed, and if so, which semver increment to use (major, minor, or patch).

### CODE REVIEW

File: docs/issues.md [hunk 1]
```diff
@@ -1,3 +1,15 @@
...
+You receive a failure when you try to deploy any IBM Cloud Databases (ICD) module that includes the autoscaling block and uses IBM Cloud Provider version 1.50 or later.
+
+### Why it's happening
+
+An issue exists with the IBM Cloud Provider version 1.50. For more information, see [issue 4339](https://github.com/IBM-Cloud/terraform-provider-ibm/issues/4339) in the Terraform Provider GitHub repo.
+
+### How do I solve this problem?
+
+- Set the `IBM-Cloud/ibm"` provider to use only [version 1.49](https://github.com/IBM-Cloud/terraform-provider-ibm/releases/tag/v1.49.0).
+- Make sure that properties that need to be set and that are passed from your deployment to the ICD module are not set to null or empty values.
```
Comment (ocofaigh) [inline docs/issues.md:15]: Hmm, I think we have a workaround for this now. But I think it still has to be rolled out to our ICD modules. Maybe we go with this for now, but update (or remove) when we roll the workaround out.
@shemau anything else to add here?

Comment (vburckhardt) [reply docs/issues.md:15]: @ocofaigh - workaround documented here is for consumer wanting to use auto-scaling capability in the module.
The workaround we discussed last week with @shemau is to prevent the issues from happening to consumer for the module who do not use auto-scaling capabilities.

Comment (SirSpidey) [reply docs/issues.md:15]: Thanks @shemau. My intent is to reword this to indicate that if you see the error, make sure that you're on the latest release of the module first. If you still have the problem, then use the workarounds about the provider version and properties.

**Comment by SirSpidey** [comment]
@shemau @ocofaigh Finally getting back to this. Please review the updates.

**Review by ocofaigh — APPROVED** [review]
